### ISSUE TITLE
XII Stag (Japan) audio issues

### ISSUE THREAD

**Issue by BPzeBanshee** [jpd002/Play-#592]
OS: Windows 7 Ultimate 64-bit
CPU: Intel i7-2600
GPU: nVIDIA GeForce GTX 1060

The game starts and otherwise runs correctly with some video stuttering that I assume is par for the course given the emulator's current state, however music does not work correctly. The stage 1 music appears to constantly loop it's first second of music and never get past that point, nor does it stop after you beat the stage or go back to the main menu.

For reference, some videos of the game in action working as intended
https://www.youtube.com/watch?v=kqnc2G9t0Ys - real hardware (camera footage), Japan version
https://www.youtube.com/watch?v=asAEejkmGOo - unsure if real or emulated (capture card footage), PAL version

**Comment by Gamemulatorer** [comment]
Duplicate of https://github.com/jpd002/Play-Compatibility/issues/173

**Comment by BPzeBanshee** [comment]
> 
> 
> Duplicate of [jpd002/Play-Compatibility#173](https://github.com/jpd002/Play-Compatibility/issues/173)

My report predates that one by nearly a year, so I'd argue theirs is a duplicate of mine on chronology. But if game-specific emulation bugs are meant to be reported there, then I'll admit fault for not seeing that section when I reported this bug 4 years ago.

That said, I understand the compatibility list has no criteria for sound. This bug I've reported is still present in 0.42-31-ge9bc7228. Are audio emulation issues with Play! considered a "wontfix" issue?
